tests/end_to_end/tests.py

```
@@ -252,6 +260,7 @@ def test_run_configuration(self):
 
         self.step_add_service_description(client_id)
         self.step_enable_service_description(client_id)
+        self.step_add_service_access_rights()
```

Can silently fail, verify transitions afterwards to remedy.